lizmap/modules/lizmap/lib/Form/QgisForm.php

```diff
@@ -546,6 +541,7 @@ public function saveToDb($feature = null)
             throw new \Exception('Save to database can\'t be done for the layer "'.$this->layer->getName().'"!');
         }
 
+        $cnx = $this->layer->getDatasourceConnection();
```

Comment:
this variable is not used below. You should not add this line of code.

Reply:
It is [there](https://github.com/alagroy-42/lizmap-web-client/blob/2d83af54836fae08eb0770b2e9e44d7b2aaca0b1/lizmap/modules/lizmap/lib/Form/QgisForm.php#L575)

Reply:
Sorry, I looked into the `check()` method, where `$cnx` is unused.